browser-test/src/support/applicant_questions.ts

```diff
@@ -473,6 +473,10 @@ export class ApplicantQuestions {
     await waitForPageJsLoad(this.page)
   }
 
+  async clickCreateAnAccount() {
+    await this.page.getByRole('button', {name: 'Create an account'}).click()
```

Comment:
(blocking) Checking auth flow should go in the `browser-test/src/support/index.ts` file.

(question) Is it possible to have multiple buttons with this label on the screen? Such as on the page and also in a modal. If so, this would fail on pages that have that. Unless Playwright has some logic to look at the higher layers I'm unaware of. Can you give it a look.

Reply:
Deleted code